doc/howto/incus_alias.md

```diff
@@ -15,6 +15,13 @@ Within the [`incus alias`](incus_alias.md) command, you can use the following su
 
 Run [`incus alias --help`](incus_alias.md) to see all available subcommands and parameters.
 
+```{note}
+_Command aliases_ are different from [_images aliases_](images.md).
```

Comment:
Should be "from {ref}`_image aliases_ <images>`" I think for the doc to generate correctly.